tasks/main.yml

```diff
@@ -88,6 +88,13 @@
     enablerepo: "rhui-microsoft-*"
   when: ansible_system_vendor == "Microsoft Corporation"
 
+- name: Force install kernel
+  package:
+    name: kernel-{{ __hpc_force_kernel_version }}
+    state: present
+  when: __hpc_force_kernel_version is not none
+  notify: Reboot system
```

Comment:
Do you need to do anything with bootloader here to ensure that the specified kernel is the one used to boot?

Reply:
not sure